ArduCopter/APM_Config.h

```diff
@@ -5,44 +5,44 @@
 //#define AC_TERRAIN            DISABLED            // disable terrain library
 //#define PARACHUTE             DISABLED            // disable parachute release to save 1k of flash
 //#define NAV_GUIDED            DISABLED            // disable external navigation computer ability to control vehicle through MAV_CMD_NAV_GUIDED mission commands
-//#define OPTFLOW               DISABLED            // disable optical flow sensor to save 5K of flash space
+#define OPTFLOW               DISABLED            // disable optical flow sensor to save 5K of flash space
 //#define FRSKY_TELEM_ENABLED   DISABLED            // disable FRSky telemetry
-//#define PRECISION_LANDING     DISABLED            // disable precision landing using companion computer or IRLock sensor
-//#define BEACON_ENABLED        DISABLED            // disable beacon support
-//#define SPRAYER_ENABLED       DISABLED            // disable the crop sprayer feature (two ESC controlled pumps the speed of which depends upon the vehicle's horizontal velocity)
+#define PRECISION_LANDING     DISABLED            // disable precision landing using companion computer or IRLock sensor
+#define BEACON_ENABLED        DISABLED            // disable beacon support
+#define SPRAYER_ENABLED       DISABLED            // disable the crop sprayer feature (two ESC controlled pumps the speed of which depends upon the vehicle's horizontal velocity)
 //#define WINCH_ENABLED         DISABLED            // disable winch support
-//#define GRIPPER_ENABLED       DISABLED            // disable gripper support
+#define GRIPPER_ENABLED       DISABLED            // disable gripper support
 //#define RPM_ENABLED           DISABLED            // disable rotations per minute sensor support
 //#define STATS_ENABLED         DISABLED            // disable statistics support
 //#define MODE_ACRO_ENABLED     DISABLED            // disable acrobatic mode support
 //#define MODE_AUTO_ENABLED     DISABLED            // disable auto mode support
 //#define MODE_BRAKE_ENABLED    DISABLED            // disable brake mode support
-//#define MODE_CIRCLE_ENABLED   DISABLED            // disable circle mode support
-//#define MODE_DRIFT_ENABLED    DISABLED            // disable drift mode support
-//#define MODE_FLIP_ENABLED     DISABLED            // disable flip mode support
-//#define MODE_FOLLOW_ENABLED   DISABLED            // disable follow mode support
+#define MODE_CIRCLE_ENABLED   DISABLED            // disable circle mode support
+#define MODE_DRIFT_ENABLED    DISABLED            // disable drift mode support
+#define MODE_FLIP_ENABLED     DISABLED            // disable flip mode support
+#define MODE_FOLLOW_ENABLED   DISABLED            // disable follow mode support
 //#define MODE_GUIDED_ENABLED   DISABLED            // disable guided mode support
-//#define MODE_GUIDED_NOGPS_ENABLED   DISABLED      // disable guided/nogps mode support
+#define MODE_GUIDED_NOGPS_ENABLED   DISABLED      // disable guided/nogps mode support
```

Comment:
We'll need to re-enable GUIDED no GPS

Reply:
That is unexpected ! I will revert.
Anything else ?

Reply:
You will need to re-enable flip as well ;-)

Reply:
that is courageous to do flip with manna drone considering the flip code !